mover-syncthing/entry.sh

```diff
@@ -133,8 +129,8 @@ for op in "$@"; do
       # ensure our environment is configured before syncthing runs
       preflight_check
 
-      # launch syncthing			
-      syncthing -home "${SYNCTHING_CONFIG_DIR}"
+      # launch syncthing
+      exec syncthing -home "${SYNCTHING_CONFIG_DIR}" --verbose
```

Comment:
Added the `exec` so that when the Pod receives a shutdown signal, it will be passed to syncthing, giving it a chance to stop cleanly